apps/website-25/src/components/courses/MultipleChoice.tsx

```diff
@@ -1,5 +1,6 @@
 import CTALinkOrButton from '@bluedot/ui/src/CTALinkOrButton';
 import React from 'react';
+import ReactMarkdown from 'react-markdown';
```

Comment:
You might want to use `MarkdownExtendedRenderer` instead now - should be a drop-in replacement.